db/seeds.rb

```diff
@@ -4,8 +4,16 @@
 service = Service.find_by url: 'localhost'
 unless service
   service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
-  item_type = FactoryGirl.create :item_type, name: 'TEST_CREATE_RENTAL_TYPE', creator: service
-  item = FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
+  item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+  another_item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE_2', creator: service
+  # now we're going to need a lot of items to test with
+  FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
```

Comment:
Side note: I know I am playing 'johnny come lately' here, but I not sure FG is the right tool for seed files.

Having seed files run in production is not unheard of. Worth discussion at some point.